modules/channel-web/src/views/lite/main.tsx

```diff
@@ -158,6 +160,25 @@ class Web extends React.Component<MainProps> {
     this.handleResetUnreadCount()
   }
 
+  /*
+    Call example:
+
+    const payload = bp.RealTimePayload.forVisitor(event.target, 'webchat.data', {
+      payload: {
+        currentUILanguage: 'ar'
+      }
+    })
+    bp.realtime.sendPayload(payload)
+  */
+  handleDataMessage = event => {
+    console.log(event.payload)
```

Comment:
Is this needed?

Reply:
It's usage example.
May be it could be better to put it to documentation, but not sure, in which section.